vtex/loaders/intelligentSearch/productListingPage.ts

```diff
@@ -143,12 +148,15 @@ export interface Props {
    * @deprecated Use product extensions instead
    */
   similars?: boolean;
-
   /**
    * @hide true
    * @description The URL of the page, used to override URL from request
    */
   pageHref?: string;
+  aditionalFieldsInQuery?: {
+    label: string,
+    value: string
+  }[]
```

Comment:
```suggestion
/** @title {{{label}}} - {{{value}}} */
interface AditionalQueryFields {
    label: string,
    value: string
}

/// ...
  aditionalFieldsInQuery?: AditionalQueryFields[]
```

By doing the form in the admin UI have a better name/title that's readable for humans

Reply:
Also, this prop was added to allow `showSponsored` and `placement`? Why not add it directly into the interface?